js-beautify/js-beautify.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for js_beautify
+// Project: https://github.com/beautify-web/js-beautify/
+// Definitions by: Josh Goldberg <https://github.com/JoshuaKGoldberg/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare var js_beautify: {
+    (js_source_text: string, options?: {
+        "indent_size"?: number;
```

Comment:
These double quotes is not required.
Could you please remove the double quotes?